src/client/common/application/applicationEnvironment.ts

```diff
@@ -41,6 +41,28 @@ export class ApplicationEnvironment implements IApplicationEnvironment {
                 return;
         }
     }
+    public get userCustomKeybindingsFile(): string | undefined {
+        const vscodeFolderName = this.channel === 'insiders' ? 'Code - Insiders' : 'Code';
```

Comment:
Could the path for this be acquired from the global storage path (so you wouldn't need special code for each OS)

Reply:
Meaning go up from the global storage path?

path.resolve will eliminate '..' in paths I believe.

Reply:
This is modeled on how we find the user settings file (defined right above this), I didn't want to deviate from that in case there was a particular reason it was implemented that way.

Reply:
Ah. Maybe whomever wrote that didn't realize they could just go up. 

This won't work when using say VS code OSS bits (the bits you build).

The path if you used the global storage path would be something like:

```
path.resolve(path.join(extensionContext.globalStoragePath, '..', '..', 'keybindings.json'))
```